src/Hng.Domain/Entities/UserRole.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace Hng.Domain.Entities
+{
+    public class UserRole:EntityBase
```

Comment:
This table is redundant. You do not need to create this link table. Specify the relationship between tables, EntityFramework creates it.

Reply:
we need it for navigation from organisations